starlette/formparsers.py

```diff
@@ -122,16 +122,15 @@ async def parse(self) -> FormData:
 
 
 class MultiPartParser:
-    max_file_size = 1024 * 1024  # 1MB
-    max_part_size = 1024 * 1024  # 1MB
-
     def __init__(
         self,
         headers: Headers,
         stream: typing.AsyncGenerator[bytes, None],
         *,
         max_files: int | float = 1000,
         max_fields: int | float = 1000,
+        max_file_size: int = 1024 * 1024,  # 1MB
+        max_part_size: int | float = 1024 * 1024,  # 1MB
```

Comment:
Should we remove defaults here and keep the default at top most function?

Reply:
This is okay.